src/com/xilinx/rapidwright/interchange/PhysNetlistReader.java

```diff
@@ -296,6 +296,11 @@ private static void readPlacement(PhysNetlist.Reader physNetlist, Design design,
                             strings.get(otherCell.getMultiCell()),
                             strings.get(otherCell.getMultiType())));
                 }else {
+                    if(c.getBEL().getPin(belPinName) == null) {
+                        System.err.println("WARNING: On cell " + c.getName() + ", a logical pin '" +
+                                c.getType() + "." + cellPinName + "' is being mapped on to a BEL pin '" 
+                                + c.getBELName() + "." + belPinName + "' that does not exist.");
+                    }

```

Comment:
```suggestion
                                + c.getBELName() + "." + belPinName + "' that does not exist. " 
                                + "This may result in an invalid design.");
                    }

```